src/helm/benchmark/static/benchmarking.js

```diff
@@ -124,6 +124,55 @@ $(function () {
     return $table;
   }
 
+  function renderPlots() {
+    const container = $('<div>', {class: "container"});
+    const links_div = $('<div>');
+    container.append(links_div);
+    const table_links = [];
+
+    function renderPlot(name, title) {
+        container.append($('<h3>').append($('<a>', {id: title}).append(title)));
+        container.append($('<img>', {src: plotUrl(suite, name), class: "img-fluid"}));
+        table_links.push($('<a>', {href: '#' + title}).append(title));
+    }
+
+    renderPlot("generic_summary", "Scenario summary per metric");
```

Comment:
Would you mind copying the caption from the paper into the Javascript so people viewing the page can have a bit more context?